data/tags.yaml

```diff
@@ -0,0 +1,21 @@
+# Config file for the tagger analyzer.
+#
+# Each definition needs to define either a query_string or a query_dsl.
+#
+# The create_view is an optional boolean that determines whether a view
+# should be created if there are hits.
+#
+# The tagger works in the way that the query is run and the emojis and
+# tags that are defined are applied to the matching events.
+# A view is created if the tag is added (only if there are results).
+#
+# Example:
+#
+# test_tagger:
+#        query_string: 'test'
+#        attribute: 'domain'
```

Comment:
OK, so this is nowhere used, and I'm not sure how you would even use it in this case....

(the only place this is used is in the tests.... and to check it is empty... so please remove it

Reply:
oops, removed this elsewhere but missed this one. Good catch :) Fixed now.